### ISSUE TITLE
raise FinamDownloadError

### ISSUE THREAD

**Issue by Celeevo** [ffeast/finam-export#32]
Добрый день!
Все прекрасно работало, но вот уже несколько дней при запросе котировок выдает ошибку:

  File "C:\Users\...\finam-export\export.py", line 118, in _parse_js_assignment
    raise FinamDownloadError('Unable to parse line: {}'.format(line))
exception.FinamDownloadError: Unable to parse line: п»ї<htmlxmlns="http://www.w3.org/1999/xhtml">

У всех так, Финам что-то поменял?

**Comment by Whisperes** [comment]
Подтверждаю аналогичную проблему.

**Comment by aielx** [comment]
Да, что-то изменилось... Скорее всего дополнительные проверки на стороне финама. Сейчас не работает
